packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -360,10 +390,34 @@ const SearchInputBase: React.FunctionComponent<SearchInputProps> = ({
     />
   );
 
-  const buildExpandableSearchInput = ({ ...searchInputProps } = {}) => (
+  const expandableToggle = (
+    <>
+      {!hasAnimations && <InputGroupItem isPlain>{singleButtonToggle}</InputGroupItem>}
+      {hasAnimations && (
+        <>
+          <InputGroupItem className={inputGroupStyles.modifiers.searchExpand} isPlain>
+            {expandToggleButton}
+          </InputGroupItem>
+          <InputGroupItem className={inputGroupStyles.modifiers.searchAction} isPlain>
+            {collapseToggleButton}
+          </InputGroupItem>
+        </>
+      )}
+    </>
+  );
+
+  const buildExpandableSearchInput = ({ ...searchInputProps }: any = {}) => (
     <InputGroup {...searchInputProps}>
-      <InputGroupItem isFill>{buildTextInputGroup()} </InputGroupItem>
-      <InputGroupItem isPlain>{expandableToggle}</InputGroupItem>
+      <InputGroupItem
+        isFill
```

Comment:
Can you remove `isFill` from anywhere `inputGroupStyles.modifiers.searchInput` is defined? It's used to set `flex-grow`, and since we may need to change the `flex-grow` behavior down the road, I've moved it to the CSS instead of being tied to the markup to avoid a breaking change later.